pages/common/opt.md

```diff
@@ -0,0 +1,15 @@
+# opt
+
+> LLVM .bc -> .bc modular optimizer and analysis printer. `opt` takes LLVM source files as input, runs the specified optimizations or analyses on it, and then outputs the optimized file or the analysis results.
+
+- Run an optimization or an analysis in a bitcode file:
```

Comment:
Perhaps

```suggestion
- Run an optimization or analysis on a bitcode file:
```

....will read better?

Also, can `file.bc` be `path/to/file.bc` please?